updating/updating_a_cluster/updating-cluster-web-console.adoc

```diff
@@ -19,49 +19,39 @@ You can update, or upgrade, an {product-title} cluster by using the web console.
-
-* Have access to the cluster as a user with `admin` privileges.
-See xref:../../authentication/using-rbac.adoc#using-rbac[Using RBAC to define and apply permissions].
-* Have a recent xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your update fails and you must restore your cluster to a previous state.
-* Support for {op-system-base}7 workers is removed in {product-title} {product-version}. You must replace {op-system-base}7 workers with {op-system-base}8 or {op-system} workers before updating to {product-title} {product-version}. Red Hat does not support in-place {op-system-base}7 to {op-system-base}8 updates for {op-system-base} workers; those hosts must be replaced with a clean operating system install.
-* Ensure all Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update. See xref:../../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Updating installed Operators] for more information.
-* Ensure that all machine config pools (MCPs) are running and not paused. Nodes associated with a paused MCP are skipped during the update process. You can pause the MCPs if you are performing a canary rollout update strategy.
-//remove this???^ or maybe just add another bullet that you can break up the update?
-* To accommodate the time it takes to update, you are able to do a partial update by updating the worker or custom pool nodes. You can pause and resume within the progress bar of each pool.
-* If your cluster uses manually maintained credentials, update the cloud provider resources for the new release. For more information, including how to determine if this is a requirement for your cluster, see xref:../../updating/preparing_for_updates/preparing-manual-creds-update.adoc#preparing-manual-creds-update[Preparing to update a cluster with manually maintained credentials].
-* Review the list of APIs that were removed in Kubernetes 1.27, migrate any affected components to use the new API version, and provide the administrator acknowledgment. For more information, see xref:../../updating/preparing_for_updates/updating-cluster-prepare.adoc#updating-cluster-prepare[Preparing to update to {product-title} 4.14].
-* If you run an Operator or you have configured any application with the pod disruption budget, you might experience an interruption during the update process. If `minAvailable` is set to 1 in `PodDisruptionBudget`, the nodes are drained to apply pending machine configs which might block the eviction process. If several nodes are rebooted, all the pods might run on only one node, and the `PodDisruptionBudget` field can prevent the node drain.
+include::modules/before-updating-web-console.adoc[leveloffset=+1]
```

Comment:
nit: these prereqs are largely orthogonal to the interface used to request the update.  Even if we don't have the command-line and other process sections include this new prereq module yet, do we want to pick a less-opinionated name like `modules/updating_a_cluster/prerequisites.adoc` and try to keep the prerequisites UI-agnostic?  Also, this is a bit sticky because starting with 4.14, we have [this whole section of pre-update things][1].

![image](https://github.com/openshift/openshift-docs/assets/209920/98e8efd5-ebbe-40f3-9379-f412d0e48414)

so if we spend particularly much time talking about prereqs in the process assemblies, we may want to push more of that discussion into prereq assemblies.

[1]: https://docs.openshift.com/container-platform/4.14/updating/preparing_for_updates/updating-cluster-prepare.html

Reply:
I'm a fan of renaming this module to something that's not specific to any update method, like `before-updating-ocp.adoc`. I feel like putting prerequisite in the name might make it confusing to differentiate between these softer considerations and the actual prereqs of the procedures.

Once that "Preparing to update" page gets filled with content other than API acks (or if new assemblies are created), I'm more than happy to help migrate some of the content out of this `before-updating-web-console.adoc` module and into that assembly. But if possible, I'd like to keep that effort separate from this current PR.